2.7/s2i/bin/assemble

```diff
@@ -15,6 +15,7 @@ function should_collectstatic() {
 # (pip script installer).
 function install_pipenv() {
   echo "---> Installing pipenv packaging tool ..."
+  pip install -U virtualenv
```

Comment:
Each of the images contains different Python version and different software collection, this is the reason why also s2i scripts have to be different in some cases. 
**python-34-centos7** image originally contains ancient versions of pip and virtualenv:
```
docker run -it centos/python-34-centos7 bash
(app-root)pip --version
pip 1.5.6 from /opt/app-root/lib/python3.4/site-packages (python 3.4)
(app-root)virtualenv --version
1.11.6
```
while **centos/python-35-centos7** image contains **pip 7.1.2** and **virtualenv 13.1.2**.
The reason why virtualenv package is updated from PyPI in 34 image is that old versions of pip and virtualenv are incompatible with code that follows. In 2.7, 3.5 and 3.6  overwriting of the virtualenv binary from collection by package from PyPI is not necessary to make Pipenv work.

Reply:
> In 2.7, 3.5 and 3.6 overwriting of the virtualenv binary from collection by package from PyPI is not necessary to make Pipenv work.

But it doesn't hurt either, and that way all containers have the same, up-to-date versions of pip and virtualenv (both of which are outdated in all other containers)

The point of this PR is to bring the versions in line  with each other. Because I'm certainly NOT going to create PR's in a repo where I have to do the _same_ change in _four_ different places _by hand_.

Reply:
I see your point, but containers should use scl content in case there is not good reason to use something else, what do you thing @hhorak?